private-ai-services/README.md

```diff
@@ -0,0 +1,12 @@
+# Private AI Services
```

Comment:
We don't have a list of Private AI Service versions on the main `README` or anywhere to link to this from. Did you want to add something there, or will it be linked from somewhere outside of the repo?

Reply:
Added a little (1-row) table below.